frontend/components/InfoBanner/InfoBanner.tsx

```diff
@@ -32,6 +35,7 @@ const InfoBanner = ({
     baseClass,
     {
       [`${baseClass}__${color}`]: !!color,
+      [`${baseClass}__border-${borderRadius}`]: !!borderRadius,
```

Comment:
why not `${baseClass}__border-radius-${borderRadius}`? `border` is also a concept so it would be nice to keep them distinct